src/test/java/de/tum/in/www1/artemis/user/UserUtilService.java

```diff
@@ -119,6 +119,45 @@ public List<User> generateAndSaveActivatedUsers(String loginPrefix, String commo
         return generatedUsers;
     }
 
+    /**
+     * creates and saves users with passed registration numbers
+     *
+     * @param registrationNumbers registration numbers of users
+     * @param testPrefix          used to set the user login
+     * @return the created users
+     */
+    public List<User> setRegistrationNumberOfUsers(List<String> registrationNumbers, String testPrefix) {
```

Comment:
```suggestion
    /**
     * Updates students starting with suffix 1 with the respective registration numbers..
     * Users have to exist to be updated.
     *
     * @param registrationNumbers registration numbers of users. Size controls the number of students getting updated.
     * @param testPrefix the prefix used for the student login
     * @return the updated users
     */
    public List<User> setRegistrationNumberOfStudents(List<String> registrationNumbers, String testPrefix) {
```
1. It's only about students
2. If a user does not exist an exception gets thrown